_mentalModels/2022-03-20-stairs.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The Stairs
+
+draft: false
+
+page : false 
+
+image: /assets/img/mental-models/stairs.svg
+
+excerpt: >
+  The Stairs is a good metaphor to think about _progression_- specially with a human factor.
+
+---
+
+<img
+  alt="A schematic stairs pattern"
+  src="/assets/img/mental-models/stairs.svg"
+  style="max-height: 480px; margin: 2rem auto"
+/>
+
+The Stairs is a good metaphor to think about _progression_ - specially with a
+human factor.
+
+It could be about the way a team project is going at work, or the typical
+learning curve of mastering a new language. It can even be about one's personal
+advancement toward a career goal.
+
+It could looks like linear at first thought. We have a starting line and a goal.
+Isn't the progression like the straight line between the two? It's certainly
+not. Taking a step back for the progression itself helps us understand the real
+pattern.
+
+This is never linear. We alternates between moments of slow and quick
+progression. It's like going up a stair with very long step.
```

Comment:
```suggestion
We alternate between moments of slow and quick
progression. It's like going up a stair where each step has a very long thread and a tall riser at the end.
```

https://www.stair-parts.com/wp-content/uploads/staircase-diagram-parts-of-a-staircase-stair-parts-com.jpg

J'arrive pas à croire que je met cette image dans une peer review.